tests/test_torch_internal_frontend.py

```diff
@@ -4,15 +4,16 @@
 
 import _setup_test_env  # noqa
 
+import torch
 import pytest
 
 from returnn.tensor import Tensor, Dim
 from returnn.torch.frontend import TorchFrontend as rf
 
 
 def test_compare_eq():
-    a_raw = rf.Tensor([2.0, 2.0, 2.0])
-    b_raw = rf.Tensor([1.0, 2.0, 3.0])
+    a_raw = torch.tensor([2.0, 2.0, 2.0])
+    b_raw = torch.tensor([1.0, 2.0, 3.0])
```

Comment:
Don't do this. Use only `rf` functions.

Reply:
I will use `convert_to_tensor`.

Reply:
For these I believe we should have `rf.constant()` as we discussed in the past, which is not available yet. If this is correct, I suggest we temporarily leave it like this.

Reply:
Just add `rf.constant` then. Or use `rf.range` or sth else. This really should not be complicated.